libbeat/common/kubernetes/metadata_test.go

```diff
@@ -70,12 +71,12 @@ func TestPodMetadataDeDot(t *testing.T) {
 				},
 			},
 			meta: common.MapStr{
-				"pod":       common.MapStr{"name": "", "uid": "005f3b90-4b9d-12f8-acf0-31020a840133"},
-				"namespace": "",
-				"node":      common.MapStr{"name": "test"},
-				"labels":    common.MapStr{"a": common.MapStr{"value": "bar", "key": "foo"}},
+				"pod":    common.MapStr{"name": ""},
+				"uid":    "005f3b90-4b9d-12f8-acf0-31020a840133",
```

Comment:
As `uid` has be already around before, we should not change anything here so this is only a note from my side.

I often use `id` instead of `uuid` or `uid` for consistency. And as long as there is only one `id` I expect it to be unique.

Reply:
`UID` is a well-known field in Kubernetes, changing it to `id` could confuse some users IMO

Reply:
ok